app/controllers/concerns/account_owned_concern.rb

```diff
@@ -4,6 +4,7 @@ module AccountOwnedConcern
   extend ActiveSupport::Concern
 
   included do
+    before_action :authenticate_user!, if: :whitelist_mode?
```

Comment:
Doesn't it break ActivityPub stuff? Plus you call that again in pretty much everything including `AccountOwnedConcern`

Reply:
Good catch